static/js/components/inputs/FieldsOfStudySelectField.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+import SelectField from './SelectField';
+import FIELDS_OF_STUDY from '../../fields_of_study';
+
+let fieldOfStudyOptions = _.map(FIELDS_OF_STUDY, (name, code) => ({
+  value: code,
+  label: name
+}));
```

Comment:
hrm, maybe it's something to do with how webpack bundles stuff? haha. maybe if we import and run the shim in this file too?